src/index.html

```diff
@@ -13,6 +13,36 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
```

Comment:
This image isn't background, so you need to add image with the img tag

Reply:
Sorry, it was confusing for me as there was this line "background-image: url() should be relative to the index.scss. So should start with ../images." in the task requirements. So I thought it should be background-image.